packages/webapp/pages/recruiter/analyze.tsx

```diff
@@ -1,5 +1,5 @@
 import type { ReactElement, ReactNode } from 'react';
```

Comment:
This file is so long. It will be easier to maintain if we split to different files

Reply:
Will adjust :)